shared/src/main/scala/kinesis4cats/producer/fs2/FS2Producer.scala

```diff
@@ -85,6 +86,17 @@ abstract class FS2Producer[F[_], PutReq, PutRes](implicit
     } yield ()
   }
 
+  /** Put records into the producer's buffer, to be batched and produced at a
+    * defined interval
+    *
+    * @param records
+    *   A [[cats.data.NonEmptyList NonEmptyList]] of
+    *   [[kinesis4cats.producer.Record Records]]
+    * @return
+    *   F of Unit
+    */
+  def putN(records: NonEmptyList[Record]): F[Unit] = records.traverse_(put)
```

Comment:
**thought:** maybe accepting any `Foldable` would be better? idk

Reply:
Sure, that works